src/pwrite.rs

```diff
@@ -97,4 +92,19 @@ impl<
         R: ?Sized + Index<usize> + IndexMut<RangeFrom<usize>> + MeasureWith<Ctx>,
```

Comment:
one thing that's puzzling me is how this blanket impl doesn't overlap the implementation of Pwrite for DynamicBuffer? (i.e., violates single implementation rule)

Reply:
`DynamicBuffer` doesn't have a range index or implement `MeasureWith`.